app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java

```diff
@@ -238,6 +239,7 @@ public void onCheckboxClicked(View view) {
                 URL url = new URL(rawUrl);
```

Comment:
change `SERVER_URL` to  `BuildConfig.WEBSERVICE_URL` to refer the webservice url